### DISCUSSION TITLE
ESMTP Name Change

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3223] [Development discussions]
Hello,

I want to change the Postal name shown in the image. How can I do this?

![image](https://github.com/user-attachments/assets/3d1e99ad-ca21-47d2-9b77-26ae70730b49)

**Comment by willpower232** [comment]
Looks like that is hardcoded and you will have to change it in the code in these places.

![image](https://github.com/user-attachments/assets/aaa9a815-5366-4517-bf60-ad8baac6deb5)

**Reply by TR-SametOzen** [reply]
Hello,

I updated these, but the old value appears again. Should I do anything extra for the update?

**Reply by willpower232** [reply]
You probably have to make a docker image with those values saved and then restart Postal

**Reply by TR-SametOzen** [reply]
> You probably have to make a docker image with those values saved and then restart Postal

How can I do the operation related to Docker?
